packages/scrollbars/src/Scrollbar.example.md

```diff
@@ -0,0 +1,101 @@
+### Basic Scrollable Example
+
+Our custom scrollbar implementation is uses the
+[react-custom-scrollbars](https://github.com/malte-wessel/react-custom-scrollbars)
+library internally.
+
+Take a look at their [API documentation](https://github.com/malte-wessel/react-custom-scrollbars/blob/master/docs/API.md)
+to view all of the available customization props and methods.
+
+```jsx
+const StyledSection = styled.div`
```

Comment:
It would be preferable to use the `MD` Typography component here (and throughout).